rpi3.mk

```diff
@@ -192,6 +192,7 @@ endif
 .PHONY: rpi3-firmware-clean
 rpi3-firmware-clean:
 	rm -f $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT)
+	rm -rf $(RPI3_STOCK_FW_PATH)
```

Comment:
Sorry, I don't have any solution, but `rm -rf` is not an option.
Hoping that no one will notice or care about the leftovers is better than `rm -rf` IMO.